tests/NonPhpFile/NetteDINeonMethodCallRenamer/config/method_call_rename_collector_configured_rule.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Nette\Tests\NonPhpFile\NetteDINeonMethodCallRenamer\Source\SecondService;
+use Rector\Renaming\Collector\MethodCallRenameCollector;
+use Rector\Renaming\ValueObject\MethodCallRename;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symplify\SymfonyPhpConfig\ValueObjectInliner;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->import(__DIR__ . '/../../../../config/config.php');
+    $services = $containerConfigurator->services();
+
+    $services->set(MethodCallRenameCollector::class)
```

Comment:
@TomasVotruba fyi this is another way how to invoke NetteDINeonMethodCallRenamer - it is done via MethoCallRenameCollector because it is possible. I don't know if you are able to disable this and tell users "Use only RectorInterface services"

Reply:
This should be configured rule instead.